open-build-service-mock/src/api/source.rs

```diff
@@ -97,14 +114,30 @@ impl Respond for PackageSourceListingResponder {
             .get(package_name)
             .ok_or_else(|| unknown_package(package_name.to_owned())));
 
+        let list_meta = match find_query_param(request, "meta")
+            .unwrap_or(Cow::Borrowed("0"))
+            .as_ref()
```

Comment:
You should be able to replace this with `find_query_param(...).as_deref()` and then match on `None | Some("0")` and `Some("1")` to avoid the juggling here

Reply:
Sweet, I didn't realize you could do that